test/aggregators/aggregators.js

```diff
@@ -47,6 +47,14 @@ describe('Aggregators', () => {
     });
   });
 
+  it('has no aggregators failing when shortName is called', () => {
+    return walkTree.then(aggregators => {
+      aggregators.forEach(aggregator => {
+        assert.doesNotThrow(_ => aggregator.shortName);
```

Comment:
when would accessing a property ever cause a throw (unless you are testing that aggregator is always defined)? or am I missing something :)